src/main/java/uk/gov/hmcts/reform/locationrefdata/camel/binder/BuildingLocation.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.locationrefdata.camel.binder;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.apache.camel.dataformat.bindy.annotation.CsvRecord;
+import org.apache.camel.dataformat.bindy.annotation.DataField;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.locationrefdata.camel.constants.LrdDataLoadConstants;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
+@Component
+@Setter
+@Getter
+@CsvRecord(separator = ",", crlf = "UNIX", skipFirstLine = true, skipField = true)
+@Builder
+@RequiredArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
```

Comment:
do we need @EqualsAndHashCode. I mean is good to have. But are putting object in map

Reply:
Not in the source code but I guess it is used in the functional tests.

Reply:
I think we should remove this.

Reply:
This can not be removed as the functional tests compare building location objects in two lists.